test/specs/platform.dom.tests.js

```diff
@@ -286,9 +286,11 @@ describe('Platform.dom', function() {
         }
       });
 
-      expect(chart).toBeChartOfSize({
-        dw: 214, dh: 350,
-        rw: 214, rh: 350,
+      waitForResize(chart, () => {
```

Comment:
initial size is not right?

Reply:
@kurkle Yes. We need two rounds to detect whether can chart resizes the parent node or not.

Reply:
I've forgotten why. It should not resize the container in this case, should it?

But its probably me just not remembering something. I'll need to review this one on a computer :)

Reply:
The original commit works perfectly with the container where size is fixed but causes issues when the container is a basic block (width=100% and height=0).

First round: resizes canvas with given aspect ratio respecting only width.

For example, the container is `div` without fixed size, so by default, it has width=100% and height=0. After the first round height of the container will be changed. As the height of the container is `0`, respecting height on this step will cause the wrong size.

Second round: resizes canvas with given aspect ratio also respecting height. In a case with a basic block, this round will not change the size.